service/api_task.go

```diff
@@ -310,14 +310,25 @@ func (as *APIServer) EndTask(w http.ResponseWriter, r *http.Request) {
 		endTaskResp.ShouldExit = true
 	}
 
-	grip.Info(message.Fields{
+	msg := message.Fields{
 		"message":     "Successfully marked task as finished",
 		"task_id":     t.Id,
 		"execution":   t.Execution,
 		"operation":   "mark end",
 		"duration":    time.Since(finishTime),
 		"should_exit": endTaskResp.ShouldExit,
-	})
+	}
+
+	if t.IsPartOfDisplay() {
+		dt, err := t.GetDisplayTask()
+		if err != nil {
+			grip.Error(errors.Wrapf(err, "can't get display task for task %s", t.Id))
```

Comment:
It's not useful to log errors to grip unless we have a message.Fields (so we know how to refer to it in the future) and a reason to refer to it in the logs in the future.